### PR TITLE
[ml-project] Miguel Ceinos

### PR DESCRIPTION
**Pull request by MdeMaic**
Muy guay el proyecto! Ganas de seguir.

### CODE REVIEW

**Comment by ferrero-felipe** [comment]
Muuuuy buen trabajo, Maic!
Antes de nada, tu `readme` está genial! Super descriptivo, interesante, llama la atencion, los graficos son la guinda de el pastel. :top:
Tu trabajo está genial, se te dá super bien el machine learning y lo has currado un monton.
Lo unico que te comento es que si haces un `jupyter` para limpieza y preparacion de datos, exportas los csv y lo haces una unica vez, sin repetirlo en todos los ficheros.
La otra cosa es que el `train_test_split` no hace siempre la misma division de los datos porque utiliza de aleatoriedad. Si quieres probar los diferentes modelos con la misma division de los datos para que la comparacion sea más verdadera, tienes que dividirles solo una vez o definir el parametro `random_state` al llamar `train_test_split` para que esta funcion use siempre el mismo `seed` de pseudo aleatoriedad.
Lo de almacenar los parametros de cada test nos es la guinda de el pastel, es toda la puta pasteleria y más! :top: :top: :top:

Enhorabuena por el proyectazo.
¡Sigue Así!

image
